Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -154,6 +154,9 @@ define([
      * Part of the {@link Cesium3DTileContent} interface.
      */
     Batched3DModel3DTileContent.prototype.initialize = function(arrayBuffer, byteOffset) {
+        // Destroy expired resources if they exist.
+        destroyResources(this);
```

Comment:
I kind of agree. The main reason I did that is the i3dm and b3dm contents resolves the `contentReadyToProcessPromise` after they have created their resources, so there was no good sync point to destroy the resources from the outside. But if I move those promises to the start of `initialize` it would be easier to handle.